slices/libsasl2-2.yaml

```
@@ -0,0 +1,10 @@
+﻿package: libsasl2-2
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libsasl2-modules-db_libs
+      - libssl3_libs
+    contents:
+      /usr/lib/*-linux-*/libsasl*.so.2*: 
```

```suggestion
      /usr/lib/*-linux-*/libsasl2.so.2*: 
```

This is safer, just to make sure it wouldn't conflict with another potential version (e.g. libsasl3)

---

this was a typo, not intentional, fixed it now